lib/step.rb

```diff
@@ -158,19 +158,10 @@ def run(dry_run: false, fail_fast: false)
           end
           next if path.blank?
 
-          annotation_contents = <<~EOS
-            <details>
-            <summary>`#{command_trimmed}` failed!</summary>
-
-            ```
-            #{@output.strip}
-            ```
-
-            </details>
-          EOS
+          annotation_title = "`#{command_trimmed}` failed!"
           annotation_type = failed? ? :error : :warning
```

Comment:
We can move this into `emit_annotation` since `status` is a class attribute that doesn't need to be in the method call, that way we don't even need to have `type` in the method signature.

Reply:
@carlocab Sounds good 👍🏻